zip-zcash-grants-fund.rst

```diff
@@ -0,0 +1,330 @@
+::
+
+  ZIP: 
+  Title: Zcash Grants Fund - ZCG
+  Owners: Matthew Green (@gguy)
+  Original-Authors: Matthew Green (@gguy)
+  Status: Draft
+  Category: Consensus Process
+  Created: 2024-03-04
+  License: MIT
+  Discussions-To: ...
+  Pull-Request: ...
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHALL", "SHALL NOT", "SHOULD", and "MAY"
+in this document are to be interpreted as described in BCP 14 [#BCP14]_ when,
+and only when, they appear in all capitals.
+
+The term "network upgrade" in this document is to be interpreted as
+described in ZIP 200 [#zip-0200]_ and the Zcash Trademark Donation and License
+Agreement ([#trademark]_ or successor agreement).
+
+The terms "block subsidy" and "halving" in this document are to be interpreted
+as described in sections 3.10 and 7.8 of the Zcash Protocol Specification.
+[#protocol]_
+
+"Electric Coin Company", also called "ECC", refers to the Zerocoin Electric
+Coin Company, LLC.
+
+"Bootstrap Project", also called "BP", refers to the 501(c)(3) nonprofit
+corporation of that name.
+
+"Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit
+corporation of that name.
+
+"Financial Privacy Foundation", also called "FPF", refers to the independent
+Cayman Islands-Incorporated non-profit organization of that name.
+
+"Section 501(c)(3)" refers to that section of the U.S. Internal Revenue
+Code (Title 26 of the U.S. Code). [#section501c3]_
+
+"Community Advisory Panel" refers to the panel of community members assembled
+by the Zcash Foundation and described at [#zf-community]_.
+
+The terms "Testnet" and "Mainnet" are to be interpreted as described in
+section 3.12 of the Zcash Protocol Specification [#protocol-networks]_.
+
+
+Abstract
+========
+
+This proposal describes a structure for the Zcash Grants Fund, to be
+enacted in Network Upgrade 6 and lasting for 2 years. This Grants Fund would consist
+of 20% of the block subsidies soley allocated to "Zcash Community Grants" for funding

```

Comment:
It's important to mention in the abstract that the funds would be managed by FPF